tests/test_language_detect.py

```diff
@@ -0,0 +1,86 @@
+from parameterized import parameterized, param
+from datetime import datetime
+
+from unittest.mock import Mock
+
+from tests import BaseTestCase
+
+from dateparser.custom_lang_detect.fasttext import detect_languages as fast_text_detect_languages
+from dateparser.custom_lang_detect.langdetect import detect_languages as lang_detect_detect_languages
+from dateparser.custom_lang_detect.fasttext import detect_languages
+from dateparser import parse
+from dateparser.date import DateDataParser
+from dateparser.search import search_dates
+
+
+class CustomLangDetectParser(BaseTestCase):
```

Comment:
@gavishpoddar what do you think about removing the class and leaving tests in individual functions? Given that they are already almost independent, I think it would make it much easier to read.

Reply:
Is the implemented mock correct?

Pl suggest